.npmignore

```diff
@@ -1,27 +1,29 @@
 # Cruft
-*.sublime-workspace
-.DS_Store
+._*
 .AppleDouble
+.DS_Store
 .LSOverride
-Icon
-._*
 .Spotlight-V100
+.tmp
 .Trashes
-Thumbs.db
-ehthumbs.db
-Desktop.ini
+*.sublime-workspace
 $RECYCLE.BIN/
-.tmp
+Desktop.ini
+ehthumbs.db
+Icon
 npm-debug.log*
+Thumbs.db
 
 # Code / build
-coverage
-node_modules
+.editor*
+.eslint*
+.travis*
 bower_components
+coverage
 demo
-test
+docs
```

Comment:
I don't think we can ignore all of docs because of how we are building docs in the main victory repo. I think we still need `docs/ecology.md` and `docs/docs.jsx`